### DISCUSSION TITLE
Modal Image In Column RelationManager

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#7381] [Help]
How To Create Modal Image In Column RelationManager?
from
->description(function(){
return $record->pic_url != null ? new HtmlString('<a href="'.$record->img_url.'" target="_BLANK" style="color:rgb(240 173 78);">View Image</a>') : '';
})

**Comment by pboivin** [comment]
I don't understand what you're trying to do... do you want to show an image in a modal when the column is clicked?

**Reply by AFSDARIF** [reply]
![image](https://github.com/filamentphp/filament/assets/10229660/43bc2bed-fad9-4c3c-aaee-29be3ccccc4a)
like this i create a href on description column, but is open new tab. How i can create a modal to view image?

**Comment by pboivin** [comment]
Maybe a column action?
https://filamentphp.com/docs/2.x/tables/columns/getting-started#cell-actions-and-urls

**Reply by AFSDARIF** [reply]
this is my code example

```
Tables\Columns\TextColumn::make('created_at')
->action(function (TransactionsTracking $record): void {
    $this->dispatchBrowserEvent('open-post-edit-modal', [
        'post' => $record->getKey(),
    ]);
})
```
but get this error
![image](https://github.com/filamentphp/filament/assets/10229660/fd21452d-66d1-4dd2-95ae-ba31c13d13a7)

**Reply by pboivin** [reply]
I'm not sure what you're trying to do now, I thought you were trying to display an image?
Have a look at this: https://filamentphp.com/docs/2.x/tables/actions#custom-content
